app/services/v1/sms/replay.rb

```diff
@@ -9,17 +11,28 @@ def initialize(from, to, body)
     @from_number = from
     @to_number = to
     @message = body.to_s.strip
+    @number_prefix = Phonelib.parse(from).country_code
+    @national_number = Phonelib.parse(from).national(false)
+    @user = User.left_joins(:phone).find_by(phone: { prefix: "+#{@number_prefix}", number: @national_number })
```

Comment:
A lot of users can have the same phone number, are you sure this is how we want to fins this user?

Reply:
I believe this is the only way to find user based on sms message. Any other way of user identification would require sending additional information in sms, which won't be user friendly :(

Reply:
Maybe we should use each participant as a guest or predefined participant; we don't want to accidentally share sensitive information.